org.openhab.binding.zigbee/README.md

```diff
@@ -153,6 +153,7 @@ The following coordinators are known to be supported.
 
 | Name and Link                                                                                                                                         | Coordinator                          | Configuration                                            | Comment                                                                                                                                                                                                                                                                                        |
 | ----------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------ | -------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Sonoff Zigbee 3.0 USB Dongle Plus (model “ZBDongle-E”) from ITead](https://itead.cc/product/zigbee-3-0-usb-dongle/)                                                                                                    | [Ember](#ember-ezsp-ncp-coordinator) | 115200bps<br>Hardware&nbsp;flow&nbsp;control<br>High RAM | Based on Silicon Labs EFR32MG21 MCU. This stick can also be repurposed as a a dedicated Zigbee Router if instead flash available alternative firmware. Firmware on this USB stick can be upgraded without additional hardware. Firmware available [here]([https://github.com/Elelabs/elelabs-zigbee-ezsp-utility](https://github.com/itead/Sonoff_Zigbee_Dongle_Firmware/tree/master/Dongle-E)).                                                                                                                                     |
```

Comment:
Please remove the comment about the router as this is not relevant to the binding.